client/app/config/transitions.js

```diff
@@ -0,0 +1,8 @@
+export const myCustomTransitionFunction = (transitionInfo) => {
```

Comment:
Perhaps a more descriptive method name?

Reply:
Also dead code. I can just remove this actually.